Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm

```diff
@@ -202,19 +202,19 @@ - (instancetype)initWithViewImpl:(WebKit::WebViewImpl&)impl
 
 - (void)mouseMoved:(NSEvent *)event
 {
-    if (_impl)
+    if (_impl && !_impl->ignoresMouseMoveEvents())
```

Comment:
Not entirely sure why these are here instead of in WebViewImpl::mouseMoved, but I guess either way is fine?

Reply:
Right. I don't mind either way. I'll move it.